### ISSUE TITLE
FEATURE: <generate diagnostics, drift_amount, drift_scatter and spike_positions plotting during running without GUI>

### ISSUE THREAD

**Issue by chongtianyifa** [MouseLand/Kilosort#803]
### Feature you'd like to see:

Hi,

Is it possible to allow diagnostics, drift_amount, drift_scatter and spike_positions plotting when running kilosort4 without GUI? In some cases, the sorting will be run on computing cluster. It will be better to generate the drifting image to record the drift during the experiment.

**Comment by Lathomas42** [comment]
They go through much of this in one of the example notebooks:
https://kilosort.readthedocs.io/en/latest/tutorials/basic_example.html

**Comment by chongtianyifa** [comment]
No, the scatter plotting of spikes used the position of best channel of their templates. It will better to plot with their original depth across time before drift correction.

**Comment by jacobpennington** [comment]
This is not easily doable right now since the plots are generated through `pyqtgraph` for better performance. Adding separate plotting utilities for cases like this is on the todo list.

**Comment by jacobpennington** [comment]
Added with v4.0.33.
